app/controllers/place_controller.rb

```diff
@@ -0,0 +1,93 @@
+require "slimmer/headers"
+
+class PlaceController < ApplicationController
+  before_filter :redirect_if_api_request
+  before_filter -> { set_expiry unless viewing_draft_content? }
+
+  INVALID_POSTCODE_ERROR = "invalidPostcodeError".freeze
+  NO_LOCATION_ERROR = "validPostcodeNoLocation".freeze
+
+  CUSTOM_SLUGS = {
+    "report-child-abuse-to-local-council" => {
+      locals: {
+        option_partial: "option_report_child_abuse",
+        preposition: "for",
+      }
+    },
+  }.freeze
+
+  def self.custom_slugs; CUSTOM_SLUGS; end
+
+  def show
+    setup_content_item_and_navigation_helpers("/" + params[:slug])
+    @edition = params[:edition]
+
+    if !postcode_provided?
```

Comment:
Sure! My initial thinking was that it would be clearer to sort this 'chronologically', so the first request would likely be without the postcode, but happy to change it!